src/components/Modal/StakingSuccessModal/index.tsx

```diff
@@ -15,7 +15,7 @@ import {
 import { BodyLg, BodySm, H5 } from "@threshold-network/components"
```

Comment:
Non-blocking:

Unused import

```suggestion
import { BodySm, H5 } from "@threshold-network/components"
```